en/contribute/README.md

```diff
@@ -15,13 +15,12 @@ All contributions have to be under the permissive [BSD 3-clause license](https:/
 ## Code style formatting
 
 PX4 uses [astyle](http://astyle.sourceforge.net/) for code formatting. Valid versions are
-* [astyle 2.05.1](https://sourceforge.net/projects/astyle/files/astyle/astyle%202.05.1/) (apt-get default)
```

Comment:
@mcharleb This doc update removes astyle 2.05.2 and adds 3.01 as per https://github.com/PX4/Firmware/pull/7819.

Is there anything else you think needs to be said here?